.gitignore

```diff
@@ -6,20 +6,26 @@
 /test/*.csv
 /test/*.err
 /test/*.spl
+/test/*.bat
+/test/Testing/
+/test/ref64/
```

Comment:
I don't think we want to ignore this either. We likely want to commit it at some point.